docs/Adding-Software-to-Homebrew.md

```diff
@@ -51,70 +51,97 @@ Making a new cask is easy. Follow the directions in [How to Open a Homebrew Pull
 
 #### Examples
```

Comment:
Just a general note that we'll have to make sure that whichever of these lands first, the other one doesn't wipe out the changes made in this file:

- https://github.com/Homebrew/brew/pull/16327